__tests__/app.test.js

```diff
@@ -187,17 +187,42 @@ describe("GET USERS", () => {
         });
       });
   });
-  test('GET:200 - comment count', () => {
+  test("GET:200 - comment count", () => {
     return request(app)
-    .get(`/api/reviews/3`)
-    .expect(200)
-    .then((commentObj)=>{
-      expect(commentObj.body.comment_count).toBe(`3`)
-    })
+      .get(`/api/reviews/3`)
+      .expect(200)
+      .then((commentObj) => {
+        expect(commentObj.body.comment_count).toBe(`3`);
+      });
   });
 });
 
-
+describe("GET Query", () => {
```

Comment:
There aren't any tests here for filtering by category, please add tests for that!